package.json

```diff
@@ -45,7 +45,8 @@
         "coverage": "jest --coverage",
         "prepare": "husky install && npm run build",
         "lint": "prettier --check .",
-        "format": "prettier --write ."
+        "format": "prettier --write .",
+        "example": "ts-node example.ts"
```

Comment:
suggestion: `ts-node` might not be installed. Using `npx ts-node example.ts` might be a better option.

Reply:
`ts-node` is being installed from our package-lock.json. I will mention it explicitly on the `devDependencies`.

Reply:
Got blocked 🤣 https://github.com/opensource-nepal/node-nepali-datetime/issues/99

Reply:
Updated